src/cmd/bootstrap.ts

```diff
@@ -132,7 +133,7 @@ export const getStoredClusterSecrets = async (
   const d = deps.terminal(`cmd:${cmdName}:getStoredClusterSecrets`)
   d.info(`Checking if ${secretId} already exists`)
   if (env.DISABLE_SYNC) return undefined
-  const kubeSecretObject = await deps.getK8sSecret(DEPLOYMENT_PASSWORDS_SECRET, env.DEPLOYMENT_NAMESPACE)
+  const kubeSecretObject = await deps.getK8sSecret(DEPLOYMENT_PASSWORDS_SECRET, 'otomi')
```

Comment:
How do we ensure that the `otomi` namespace exists ?

Reply:
the installer creates it, and would error out otherwise

Reply:
but I added the step for cli